script/RumpelConfig.sol

```diff
@@ -201,6 +213,42 @@ library RumpelConfig {
 
         return configs;
     }
+
+    // Mellow Re7 ----
+    function getMellowRe7GuardProtocolConfigs() internal pure returns (ProtocolGuardConfig[] memory) {
+        ProtocolGuardConfig[] memory configs = new ProtocolGuardConfig[](2);
+
+        // Mellow LRT Vault re7LRT
+        configs[0] = ProtocolGuardConfig({target: MAINNET_RE7LRT, allowedSelectors: new bytes4[](2)});
+        configs[0].allowedSelectors[0] = IMellow.deposit.selector;
+        configs[0].allowedSelectors[1] = IMellow.registerWithdrawal.selector;
+
+        // Mellow WBTC Vault re7RWBTC
+        configs[1] = ProtocolGuardConfig({target: MAINNET_RE7RWBTC, allowedSelectors: new bytes4[](2)});
+        configs[1].allowedSelectors[0] = IMellow.deposit.selector;
+        configs[1].allowedSelectors[1] = IMellow.registerWithdrawal.selector;
+
+        return configs;
+    }
+
+    function getMellowRe7GuardTokenConfigs() internal pure returns (TokenGuardConfig[] memory) {
+        TokenGuardConfig[] memory configs = new TokenGuardConfig[](3);
+
+        configs[0] = TokenGuardConfig({token: MAINNET_WBTC, allowTransfer: true, allowApprove: true});
+        configs[1] = TokenGuardConfig({token: MAINNET_RE7LRT, allowTransfer: true, allowApprove: false});
+        configs[2] = TokenGuardConfig({token: MAINNET_RE7RWBTC, allowTransfer: true, allowApprove: false});
+
+        return configs;
+    }
+
+    function getMellowRe7ModuleTokenConfigs() internal pure returns (TokenModuleConfig[] memory) {
+        TokenModuleConfig[] memory configs = new TokenModuleConfig[](2);
+
+        configs[0] = TokenModuleConfig({token: MAINNET_RE7LRT, blockTransfer: true, blockApprove: true});
+        configs[1] = TokenModuleConfig({token: MAINNET_RE7RWBTC, blockTransfer: true, blockApprove: true});
```

Comment:
<img width="739" alt="Screenshot 2024-09-27 at 12 36 29 PM" src="https://github.com/user-attachments/assets/19774f62-0379-4f0f-8b4b-e0d78c7a4a25">

Yea, the block here was based at kenton's request. I have a similar feeling re: blocking all the other assets, but also err towards minimal blocking unless we have to for now

Reply:
ok makes sense

Reply:
noticed reading that message again that kenton asked to turn disable asset locking for the receipt tokens, so i've just added this https://github.com/sense-finance/rumpel-wallet/pull/14/commits/69ef67577fac3a1cbfb9fdd91fd03d8a65909ea0 wdyt?

Reply:
I think that works. It would never make sense for the vault shares to be reward tokens, as that would re-distribute the underlying tokens in the vault in a very odd way, so I think we can be as confident as we can be that they would not be a reward token, and therefore we'd never need to transfer them from the module.

It does feel a little odd to keep updating the script so it would work with the initial configs we used. Since we already ran it with that data, updating would really only be for a redeploy, and updating the script that was used for previous executions makes it someone challenging to see the script / code we actually used to execute those initial configurations.

We might want to think about how we version these scripts for traceability. Maybe any version we submit a large config change from we have a tag for, and in docs we trace those tags to the review.

Reply:
> I think that works. It would never make sense for the vault shares to be reward tokens, as that would re-distribute the underlying tokens in the vault in a very odd way, so I think we can be as confident as we can be that they would not be a reward token, and therefore we'd never need to transfer them from the module.

👍 

> We might want to think about how we version these scripts for traceability. Maybe any version we submit a large config change from we have a tag for, and in docs we trace those tags to the review.

Oh hmm with this change https://github.com/sense-finance/rumpel-wallet/commit/7f1f19946f2c80a1a0494cf7d7d67e7df2a0b95a#diff-8ffd9e34c79698015a7836a95eb96407ba8239ca754c7ba1885a2ddf5b4c5245R147 i was trying to do that by adding tagging to the config. Each time we push an update it's independent code keyed off a tag. Was that part of what you were thinking?

Reply:
I think that aspect of it is very slick / clean.

I'm thinking of a case where we want to know the exact script we ran to setup the initial token config, that could be slightly hard w/o some tracking / tagging of previous branches. it's probably unlikely we'd need or want to do this, but would be clean for tracking of how we execute these updates.